HOMEWORKS/Chapter_03/Saeid/Homework_02/assets/js/script.js

```diff
@@ -0,0 +1,134 @@
+    "<p> . Prozessor: Intel Core i7-8265u quad<br>" +
+    "Besonderheiten: Ein Laptop der sich Ihnen anpasst - Nutzen Sie den Spectre x360 in 4 Modi (Laptop-,<br>" +
+    "Tablet-, Stand-, Tent-Position). Leicht konvertierbar, robust und mobil<br>" +
+    "Elegnantes, leichtes und hochwertig verarbeitetes Unibody Gehäuse aus Aluminium. Beleuchtete Tastatur,<br>" +
+    "integrierter Fingerabdruckleser, IR Webcam; Grafikkarte. Nvidia GeForce MX150 (2GB DDR5 dediziert)<br>" +
+    "Durch aktuelle Hochleistungshardware sind Sie für anspruchsvolle Anwendungen gerüstet und kommen mit<br>" +
+    "einer Akkuladung durch den Tag<br>" +
+    "Lieferumfang: HP Spectre x360 15-df0106ng (15,6 Zoll/UHD IPS Touch) Convertible Laptop in poseidon blau" +
+    "</p>";
+  contentOfMainCard.appendChild(textOfContent);
+  let btnOfContent = document.createElement("div"); // div of content btn
+  btnOfContent.setAttribute("class", "co_price");
+  btnOfContent.innerHTML +=
+    "<h4><strong> Our price :</strong><span>EUR 1,699.00</span></h4>" +
+    "<p><button>Add to basket</button></p>" +
+    "<span id='new_price'>New price</span>";
+  contentOfMainCard.appendChild(btnOfContent);
+
+  mainCardOfCart.appendChild(contentOfMainCard);
+
+  let tableOfMainCard = document.createElement("div"); // div of table in main card
+  tableOfMainCard.setAttribute("class", "specification");
+  tableOfMainCard.innerHTML +=
+    '<table><tr><td class="first_td">Shipping </td><td class="second_td">Prime</td></tr><tr><td class="first_td">Sold By </td><td class="second_td">Amazon.de</td></tr><tr><td class="first_td">Computer Memory Size </td><td class="second_td">16 GB</td></tr><tr><td class="first_td">Processor (CPU) Model </td><td class="second_td">Core i7</td></tr><tr><td class="first_td">Processor (CPU)<br> Manufacturer </td><td class="second_td">Intel</td></tr><tr><td class="first_td">Display Size </td><td class="second_td">15.6 inches</td></tr><tr><td class="first_td">Graphics Description </td><td class="second_td">NVIDIA GeForce MX150</td></tr><tr><td class="first_td">Hard disk Description </td><td class="second_td">SSD</td></tr><tr><td class="first_td">Hard Disk Size </td><td class="second_td">512 GB</td></tr><tr><td class="first_td">Item Weight </td><td class="second_td">2.18 kg</td></tr><tr><td class="first_td">Operating System </td><td class="second_td">Windows 10 Home 64</td></tr></table>';
+  mainCardOfCart.appendChild(tableOfMainCard);
+  // End of main
+
+  // Start footer of our shopping cart
+  let footerOfCart = document.createElement("div"); // Create a div for Card
+  footerOfCart.setAttribute("class", "footer"); // Create attribute of class
+  // End footer
+
+  cardOfContainer.appendChild(headerOfCart);
+  cardOfContainer.appendChild(mainCardOfCart);
+  cardOfContainer.appendChild(footerOfCart);
+  document.body.appendChild(cardOfContainer);
+};
+divOfContainerAndCard();
+
+// We send new price value
+let oldDate = new Date(2019, 4, 30);
```

Comment:
It can be inside updatePrice function and use parameter to make a generic function.
Also those variables will removed after finishing of function execution which save you memory resources. But still a right solution.